web/app/js/components/MetricsTable.jsx

```diff
@@ -36,11 +37,10 @@ const formatTitle = (title, tooltipText) => {
 
 const meshedColumn = {
   title: formatTitle("Meshed"),
-  dataIndex: "pods",
   key: "pods",
   className: "numeric",
   sorter: (a, b) => numericSort(a.pods.totalPods, b.pods.totalPods),
-  render: p => p.meshedPods + "/" + p.totalPods
+  render: d => d.unmeshed ? "unmeshed" : d.pods.meshedPods + "/" + d.pods.totalPods
```

Comment:
I am wondering if there are other ways to express this field. Could we use N/A, or a `?`. I also thought about 0/0 buy that doesn't make much sense. WDYT?

Reply:
It might be a lot of work, but in theory I think we could produce a count of unmeshed pods based on the tap data, and then display this field as `0/n`. That would more closely match what we're doing in the outbound table too. But like I said, I haven't investigated too thoroughly to see how hard that would be to implement base on the way that we're currently aggregating the tap data.

Reply:
Okay! I've updated the implementation to count the number of pods, and display that count in the table. I've also updated it to filter out the sources that aren't of the resource type currently being displayed in the table, so the column heading will still be correct.